setup.py

```diff
@@ -13,7 +13,7 @@ def get_requirements(env=""):
 
 setup(
     name='fastapi_websocket_rpc',
-    version='0.1.24',
+    version='0.1.25',
```

Comment:
Don't bump the version, we'll do it as part of the release process

Reply:
Hi @roekatz, is the next release in the works? We'd love to use the new pydantic support in production

Reply:
Hi @ff137 , I've actually already released version 0.1.25 of `fastapi_websocket_rpc`.
You probably need a new version of `fastapi_websocket_pubsub` as well? I'll release it now

Reply:
@ff137 Released `fastapi_websocket_pubsub-0.3.8` as well :)
LMK if that works

Reply:
@roekatz aah, I missed it! Because I was checking out the github page. Releases shows:
```
v0.1.23
Latest
on Aug 24, 2022
```
But yes, PyPi is on 0.1.25 :+1:

Similarly the pubsub repo also doesn't have tagged releases synced with the PyPi packages. Not a problem, I guess it just helps with changelogs. 

But yeah, I see pubsub on pypi is still on 0.3.7. The package probably still needs to be published.
Not too urgent for us though -- we still have some more prep work before we can upgrade other things to pydantic 2 as well
So, thanks for following up! I appreciate it

Reply:
@ff137 Oh you're right, accidentally released pubsub 0.3.8 to testpypi instead of pypi.
Now it's released to the pypi as well :)

Thanks for pointing out the missing releases in Github, I'll see if we want to fix that

Reply:
@roekatz Thanks!